azure-pipelines.yml

```diff
@@ -10,25 +10,49 @@
 strategy:
   matrix:
     linux:
-      imageName: 'ubuntu-latest'
+      imageName: 'MMSUbuntu20.04TLS'
     windows:
-      imageName: 'vs2017-win2016'
+      imageName: 'MMS2019TLS'
 
 pool:
+    name: '1ES-Hosted-AzFunc'
```

Comment:
In some cases, the VM Image may actually be ignored, so you may need to replace the line after this. See [this](https://github.com/Azure/azure-functions-core-tools/pull/2883/files) PR for reference.

Reply:
As per our conversation, let wait until @eliaslopezgt merge the change into the other repros and then we can make the change here. I have opened issue https://github.com/Azure/azure-functions-powershell-worker/issues/730 to track this.